wgsl/index.bs

```diff
@@ -316,19 +314,75 @@ However, UTF-8 is always a valid encoding for a [SHORTNAME] program.
 Note: The intent of promoting UTF-8 like this is to simplify interchange of [SHORTNAME] programs
 and to encourage interoperability among tools.
 
+[SHORTNAME] program text consists of a sequence of characters, grouped into contiguous non-empty sets forming:
+* [=comments=]
+* [=tokens=]
+* [=blankspace=]
+
+<dfn>Blankspace</dfn> is any combination of one or more of the following characters:
+* space
```

Comment:
FYI. C++ 14 allows the same set, except that it does not have carriage-return.

2.3 "Character sets"

> The basic source character set consists of 96 characters: the space character, the control characters representing horizontal tab, vertical tab, form feed, and new-line, plus the following 91 graphical characters ...